### PR TITLE
Iot-cc-ranked ecr image

### PR DESCRIPTION
**Pull request by mikev**
Create a k8s Python Spark job which ranks data rewards by country code (CC)

Input is bronze/iot_gateways_geo/
Input is silver/iot-reward-share/
output is foundation-iot-metrics/iot-cc-ranked.parquet

k8s linked update
https://github.com/helium/helium-foundation-k8s/pull/168

### CODE REVIEW

File: iot-cc-ranked/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#RUN cp wildfly-openssl-1.0.7.Final.jar /opt/spark/jars
+#RUN mv wildfly-openssl-1.0.7.Final.jar /usr/local/lib/python3.10/dist-packages/pyspark/jars/
+#RUN wget https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-bundle/1.12.262/aws-java-sdk-bundle-1.12.262.jar
+#RUN cp aws-java-sdk-bundle-1.12.262.jar /root/.ivy2/jars
+#RUN cp aws-java-sdk-bundle-1.12.262.jar /opt/spark/jars
+#RUN mv aws-java-sdk-bundle-1.12.262.jar /usr/local/lib/python3.10/dist-packages/pyspark/jars/
+#RUN wget https://repo1.maven.org/maven2/org/antlr/antlr4-runtime/4.9.3/antlr4-runtime-4.9.3.jar
+#RUN cp antlr4-runtime-4.9.3.jar /root/.ivy2/jars
+#RUN cp antlr4-runtime-4.9.3.jar /opt/spark/jars
+#RUN mv antlr4-runtime-4.9.3.jar /usr/local/lib/python3.10/dist-packages/pyspark/jars/
```
Comment (deasydoesit) [inline iot-cc-ranked/Dockerfile]: Is the commented out code needed?

File: iot-cc-ranked/build.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -x
+
+docker build -t iot-cc-ranked:latest .
```
Comment (deasydoesit) [inline iot-cc-ranked/build.sh]: How does `latest` here jive with the image versioning used in `helium-foundation-k8s`?

https://github.com/helium/helium-foundation-k8s/blob/55bf33c081ea73863a262b7a10f357822cb2c92f/manifests/poc-data-cluster/prod/helium/iot-cc-ranked.yaml#L25

File: iot-cc-ranked/run.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -x
+
+docker run -it --env-file=awskeys.env iot-cc-ranked:latest
```
Comment (deasydoesit) [inline iot-cc-ranked/run.sh]: Same comment as above

File: iot-cc-ranked/Dockerfile (hunk 1 above)
Comment (mikev) [reply iot-cc-ranked/Dockerfile]: Commented code removed.

File: iot-cc-ranked/build.sh (hunk 2 above)
Comment (mikev) [reply iot-cc-ranked/build.sh]: Fixed

File: iot-cc-ranked/run.sh (hunk 3 above)
Comment (mikev) [reply iot-cc-ranked/run.sh]: Fixed

File: iot-cc-ranked/run.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -x
+
+docker run -it --env-file=awskeys.env iot-cc-ranked:0.0.3
```
Comment (deasydoesit) [inline iot-cc-ranked/run.sh]: Does this need to be `0.0.4` too?

**Review by deasydoesit — APPROVED** [review]

**Pull request merged**